File: app/src/main/java/xyz/drean/controlbook/pojo/Student.java
package xyz.drean.controlbook.pojo;

public class Student {

    private String id;
    private String name;
    private String lastname;
    private String dni;
    private String idClassRom;

    public Student() {
    }

    public Student(String id, String name, String lastname, String dni, String idClassRom) {
        this.id = id;
        this.name = name;
        this.lastname = lastname;
        this.dni = dni;
        this.idClassRom = idClassRom;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getDni() {
        return dni;
    }

    public void setDni(String dni) {
        this.dni = dni;
    }

    public String getIdClassRom() {
        return idClassRom;
    }

    public void setIdClassRom(String idClassRom) {
        this.idClassRom = idClassRom;
    }
}

File: app/src/main/java/xyz/drean/controlbook/pojo/Student.kt
package xyz.drean.controlbook.pojo

class Student {

    var id: String? = null
    var name: String? = null
    var lastname: String? = null
    var dni: String? = null
    var idClassRom: String? = null
    var idParent: String? = null

    constructor() {}

    constructor(id: String, name: String, lastname: String, dni: String, idClassRom: String, idParent: String) {
        this.id = id
        this.name = name
        this.lastname = lastname
        this.dni = dni
        this.idClassRom = idClassRom
        this.idParent = idParent
    }
}
